Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -37,22 +38,25 @@ public sealed class Server : ServerBase, IDisposable {
         internal readonly ParseQueue _parseQueue;
         private readonly Dictionary<IDocument, VolatileCounter> _pendingParse;
         private readonly VolatileCounter _pendingAnalysisEnqueue;
+        private readonly RestTextConverter _textConverter = new RestTextConverter();
```

Comment:
If this is not stateful, can we have a static instance? If it is stateful, we should instantiate it when needed (it looks cheap enough).

Reply:
Technically it has states as it is converting. Not that it is called from multiple threads concurrently, but theoretically it could.

Reply:
Okay, let's instantiate it for each conversion, just to be safe.